future-apps/payroll/contracts/Payroll.sol

```diff
@@ -751,7 +745,7 @@ contract Payroll is EtherTokenConstant, IForwarder, IsContract, AragonApp {
      * @param _employeeId Employee's identifier
      * @return True if the employee's end date has not been reached yet, false otherwise
      */
-    function _isEmployeeActive(uint256 _employeeId) internal returns (bool) {
+    function _isEmployeeActive(uint256 _employeeId) internal view returns (bool) {
```

Comment:
Oof, we should double check if anything else is missing these modifiers, I'm surprised we haven't caught the compiler warnings yet.